modules/rosa-upgrading-cli-tutorial.adoc

```diff
@@ -0,0 +1,51 @@
+:_module-type: PROCEDURE
+//Specify the system-module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-upgrading.adoc
+
+[id="rosa-upgrading-cli_{context}"]
+= Upgrading with the rosa CLI
+
+[role="_abstract"]
+There are three methods that can be used to upgrade {product-title} clusters. This procedure describes upgrading using the `rosa` CLI.
+
+This method schedules the cluster for an immediate upgrade, if a more recent version is available.
+
+.Procedure
+. To verify that an upgrade is available, enter the following command:
++
+[source,terminal]
+----
+$ rosa list versions
+----
++
+A list displays the current version of your cluster, indicated by `true`, and a list of available versions. The following example shows that the current version is `v4.6.8`, and the most recent version is `v4.6.12`.
```

Comment:
That's not what the ```true``` means.  True means that that is the version that would be used if the user created a cluster without specifying a version.  It is the default version.

To find out what version an existing cluster is running, run rosa describe cluster.

Reply:
Fixed. Added a step for `rosa describe cluster` and corrected the verbiage for `true` being the default version.